contracts/AuctionBase.sol

```diff
@@ -90,11 +76,16 @@ contract AuctionBase is Pausable {
     return auction.allowed[bidder];
   }
 
-  /// TODO
-  // function withdrawBalance() external {
-  //   require(msg.sender == owner);
-  //   msg.sender.transfer(this.balance);
-  // }
+  // TODO
+  // @dev Allow people to withdraw their balances
+  function withdrawBalance(uint256 _id) external {
+    require(msg.sender == owner);
+    msg.sender.transfer(this.balance);
+
+    // Get auction from _id
+    Auction storage auction = auctions[_id];
```

Comment:
It seems like we should be using `memory` rather than `storage` 
re:  http://solidity.readthedocs.io/en/develop/frequently-asked-questions.html#what-is-the-memory-keyword-what-does-it-do